pkgs/top-level/python-packages.nix

```diff
@@ -4769,6 +4769,7 @@ in {
 
   ledger = (toPythonModule (pkgs.ledger.override {
     usePython = true;
+    boost = pkgs.boost179; # Current default boost (1.77) doesn’t work with Python 3.10.
```

Comment:
This should be removed once https://github.com/NixOS/nixpkgs/pull/175694 is merged.

Reply:
done in edfc49fbcf9e0884754f8a2cc406ba731e29baff